wagtail/users/views/users.py

```diff
@@ -44,7 +44,8 @@
 
 # Typically we would check the permission 'auth.change_user' (and 'auth.add_user' /
 # 'auth.delete_user') for user management actions, but this may vary according to
-# the AUTH_USER_MODEL setting
+# the AUTH_USER_MODEL setting. These are no longer used in the codebase in favour
+# of ModelPermissionPolicy, but are kept here for backwards compatibility.
 add_user_perm = f"{AUTH_USER_APP_LABEL}.add_{AUTH_USER_MODEL_NAME.lower()}"
 change_user_perm = "{}.change_{}".format(
     AUTH_USER_APP_LABEL, AUTH_USER_MODEL_NAME.lower()
```

Comment:
We could remove these, but I found code using them in the wild:

- https://github.com/borisbrue/wagtail-tenants/blob/dc18a31cf304862a556ed1d54e25fb28a918d9e7/wagtail_tenants/users/views/users.py#L19-L21
- https://github.com/uktrade/directory-cms/blob/15640637594011ff65ed460bb66d6824248b3972/users/views.py#L12